react/graphql/getSpecificationsValues.gql

```diff
@@ -0,0 +1,6 @@
+query GetSpecificationsValues($fieldId: ID){
+    fieldValues(fieldId: $fieldId) @context(provider: "vtex.catalog-graphql"){
+    fieldValueId
+    value
```

Comment:
For some reason, this file wasn't linted.

```suggestion
      fieldValueId
      value
```

Reply:
I remove all of the extra tabs and commit the updated file.